City-of-Helsinki/drupal-hdbt
# UHF-4528: Add template for minimal display on TPR-units

# Add a way to list large amounts of units on a listing [UHF-4528](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4528) & [UHF-4532](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4532) 
Make it possible for the content producers to tag and list large numbers of units at the same time (for example daycare units).

## What was done
* Added template for the minimal display on units.

## How to install
* Check from the PR https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/279

## How to test
* Check from the PR https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/279

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/279
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/116
